app/models/proxy_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ProxyType
+  include ActiveAttr::Model
+
+  attribute :id
+  attribute :name
+  attribute :url_mask
```

Comment:
I see that we are not providing "selected" as an attribute here...is the idea that UI will make 2 calls, 1. to get the list and 2. to get the current root-proxy selection to be able to highlight the current selection in the drop-down?

Reply:
That was my impression from conversation with @sadatay and @cowboyd. I can go either way.